tutorials/sdf_to_usd_arms.md

```diff
@@ -0,0 +1,97 @@
+# Convert a SDF to a USD
+
+Download the panda model from Fuel
+
+```bash
+ign fuel download -u "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/panda with ignition position controller model"
+```
+
+Copy the empty world `empty.sdf` and create a file called `panda.sdf` and add these lines inside the `<world>` tag:
+
+```xml
+<include>
+  <pose>0 0 0 0 0 0</pose>
+  <name>panda</name>
+  <uri>https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/panda with ignition position controller model</uri>
+</include>
+```
+
+Run the converter:
+
+```bash
+usdConverter panda.sdf panda.usd
+```
+
+Open IsaacSim and load the `panda.usd` file.
+
+Add to the `panda` prim the following:
+
+ - Create -> Isaac -> ROS -> Clock
+ - Create -> Isaac -> ROS -> Joint State
+    - Set *targetPrims* to `/fuel/panda`
+ - Create -> Isaac -> ROS -> Pose tree
+    - Set *articulationPrim* to `/fuel/panda`
+
+Now run the simulation.
+
+```bash
+source /opt/ros/<rosdistro>/setup.bash
+```
+
+You can use the following python script to move the robot:
```

Comment:
Can some instructions be added after the python script on how to use it? For example, does the script need to be placed in a workspace somewhere? Does the workspace need to be built? I think these would be good things to explain